dev-manuals/illustrator/Tile.md

```diff
@@ -0,0 +1,51 @@
+# How to make a Tile 
```

Comment:
Please rename this file to one of `tile.md`, `draw-tile.md`, `draw-one-tile.md`, or `draw-a-tile.md`.